src/main/proto/README.md

```diff
@@ -16,3 +16,11 @@ To rebuild the protos, open a terminal where the current working directory is th
 $> cd src/main/proto
 $> protoc --java_out=../java/ marathon.proto
 ```
+
+Finally, edit `Protos.java` and add `@SuppressWarnings("deprecation") ` before `public final class Protos`, like so:
+
+```
+@SuppressWarnings("deprecation") public final class Protos {
```

Comment:
oh it would be wonderful if this was done by a protoc plugin.

But yeah, agree it would be overkill 🗡 :)

Reply:
Hm, this will be a longer discussion I'm afraid. It should not block this PR. In any case, @bbannier mentioned several times that we should not check in neither the generated class files nor the protos but rather download them as part of our dependencies. In general I agree with him. However, I don't know how we should model this dependency.

Reply:
It'd be neat if the protos were published.

Reply:
https://issues.apache.org/jira/browse/MESOS-9095

Reply:
🙇